android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1206,38 +1193,6 @@ public void focusSearchBox() {
         }
     }
 
-    // Sets NTP background
-    public void setBackground(Bitmap bgWallpaper) {
-        CompositorViewHolder compositorView = findViewById(R.id.compositor_view_holder);
-        if (compositorView != null) {
-            ViewGroup root = (ViewGroup) compositorView.getChildAt(1);
-            if (root != null && root.getChildAt(0) instanceof FrameLayout) {
-                FrameLayout frameLayout = (FrameLayout) root.getChildAt(0);
-                DisplayMetrics displayMetrics = new DisplayMetrics();
-                getWindowManager().getDefaultDisplay().getMetrics(displayMetrics);
-                int mDeviceHeight = displayMetrics.heightPixels;
-                int mDeviceWidth = displayMetrics.widthPixels;
-                Glide.with(this)
```

Comment:
Was it some kind of animation that we don't need anymore?

Reply:
There is the issue regarding background images, both sponsored and regular, are being stretched. So we found out that we don't even need to set background image to framelayout. Setting background image to imageview in BraveNewTabPageLayout is enough

Reply:
Ok, sounds good